goth/runner/probe/mixin.py

```diff
@@ -253,6 +257,7 @@ async def pay_invoices(
                 remaining_amount=0,
                 make_deposit=True,
                 timestamp=datetime.now(timezone.utc),
+                payment_platform=self.payment_config.platform_string,
```

Comment:
I don't understand why we didn't need this before and we need it now.
I don't think I need to understand, just noting that I can't really review this.

Reply:
Without this parameter, the allocation will be created using `yagna`'s default payment platform, which currently is `zksync-rinkeby-tglm`. Before adding multi-platform support we didn't need this, as the default platform aligned with `goth`'s *only* payment platform.